### PR TITLE
 Make DNS changes introduced in 6386bc8 backward compatible

### PR DESCRIPTION
**Pull request by nvgoldin**
Commit 6386bc8 relays on DNS records stored in the Prefix metadata.
This commit ensures that environments which were initialized prior
to that commit, will work as before.

### CODE REVIEW

File: lago/virt.py [hunk 1]
```diff
@@ -507,3 +507,10 @@ def get_snapshots(self, domains=None):
             snapshots[vm_name] = vm._spec['snapshots']
 
         return snapshots
+
+    def get_compat(self):
```
Comment (gbenhaim) [inline lago/virt.py]: Good idea!

File: lago/providers/libvirt/network.py [hunk 2]
```diff
@@ -227,24 +228,48 @@ def make_ipv4(last):
...
+            if self.is_management():
+                domain_xml = ET.Element(
+                    'domain',
+                    name=self._spec['dns_domain_name'],
+                    localOnly='yes'
                 )
-            )
+                net_xml.append(domain_xml)
+                net_xml.append(
+                    self._generate_main_dns(self._spec['dns_records'], subnet)
```
Comment (gbenhaim) [inline lago/providers/libvirt/network.py]: We don't want to forward plain names ?

Comment (nvgoldin) [inline lago/providers/libvirt/network.py]: I think what makes sense is:
For all management networks - don't forward unqualified names, as we know this is the last endpoint and we probably don't want any unqualified DNS records outside the virtual network.
For all none-mgmt networks - forward(as they don't have DNS).

Prior to 6386bc8 we had forwardPlainNames='yes' in all networks(hardcoded in the template).

Comment (gbenhaim) [inline lago/providers/libvirt/network.py]: +1

**Comment by nvgoldin** [comment]
ci merge please

**Pull request merged**